tensorly/backend/mxnet_backend.py

```diff
@@ -1,41 +1,41 @@
 try:
     import mxnet as mx
+    from mxnet import numpy as np
 except ImportError as error:
-    message = ('Impossible to import MXNet.\n'
+    message = ('Cannot import MXNet.\n'
                'To use TensorLy with the MXNet backend, '
                'you must first install MXNet!')
     raise ImportError(message) from error
 
-import math
 import warnings
-
 import numpy
-from mxnet import nd
-from mxnet.ndarray import reshape, dot, transpose, stack
-
 from .core import Backend
 
+mx.npx.set_np()
+
 
 class MxnetBackend(Backend):
     backend_name = 'mxnet'
 
     @staticmethod
     def context(tensor):
-        return {'ctx': tensor.context, 'dtype': tensor.dtype}
+        return {'dtype': tensor.dtype}
 
     @staticmethod
-    def tensor(data, ctx=mx.cpu(), dtype=numpy.float32):
+    def tensor(data, dtype=None):
         if dtype is None and isinstance(data, numpy.ndarray):
             dtype = data.dtype
-        return nd.array(data, ctx=ctx, dtype=dtype)
+        elif dtype is None:
+            dtype = np.float64
```

Comment:
Should we just not set anything? In NumPy the [default is None](https://numpy.org/doc/stable/reference/generated/numpy.array.html#numpy.array) and in that case the best dtype is used, it seems that [MXNet](https://mxnet.apache.org/versions/master/api/python/docs/api/np/generated/mxnet.np.array.html#mxnet.np.array) also has a default to None.